ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml

```diff
@@ -0,0 +1,30 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    requests:
+      cpu: "3"
+      memory: 1Gi
+tests:
+- artifact_dir: /tmp/artifacts
+  as: verify
+  commands: |
+    mkdir -p /tmp/artifacts
+    make verify > /tmp/artifacts/verify.log
```

Comment:
don't think we should redirect this to a file here ... just run `make verify`

Reply:
updated in current push